java-sdk/pom.xml

```diff
@@ -29,20 +80,20 @@
     <dependencies>
         <dependency>
             <groupId>com.typesafe.akka</groupId>
-            <artifactId>akka-http_2.11</artifactId>
-            <version>10.0.3</version>
+            <artifactId>akka-http_2.12</artifactId>
+            <version>10.0.5</version>
```

Comment:
I thought we agreed to use Scala 2.11? There are several libs I need to use that aren't in 2.12.